### ISSUE TITLE
Feature request: almost correct cards

### ISSUE THREAD

**Issue by iopq** [lakesare/memcode#183]
One of the harder things in Korean is to learn where the spaces go. Even Koreans sometimes write without spaces at all.

I fail some of my cards completely because I didn't put a space in a place where it must appear. Maybe for certain things like spaces and apostrophes it could just warn you that you're "almost there" to remind you to put the space in. It means you know the word, but forgot the exact rule for what is a word and what isn't.

**Comment by lakesare** [comment]
Hm. From what I gather from briefly reading about Korean, spaces are only used to make reading the text easier, and they are not grammaticaly necessary, is that not true?

___

Either way - we have the "alternative words" feature:

<img width="450" alt="image" src="https://user-images.githubusercontent.com/7578559/211146289-ae90a265-2662-468a-9620-af893105451f.png">

How about if you answer with **answerinkorean**, it marks your answer as correct; however shows you the right spacing *answer in korean*?  
It works differently now (just marks your answer as correct), but it's easy to make it so that it shows you the right answer.

**Comment by iopq** [comment]
No, while a lot of Koreans ignore spacing, there's official spacing rules in Korean

https://korean.stackexchange.com/questions/103/when-should-spaces-be-used-between-words

**Comment by lakesare** [comment]
### Price assignment (re: [memcode#requesting-a-feature](https://github.com/lakesare/memcode#requesting-a-feature))

**Implementing almost-correct cloze-deletion cards**: **$60**
